src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -4,7 +4,9 @@ package com.amazon.ion.impl.bin
 
 import com.amazon.ion.*
 import com.amazon.ion.impl.*
+import com.amazon.ion.impl.bin.IonEncoder_1_1.*
 import com.amazon.ion.impl.bin.IonRawBinaryWriter_1_1.ContainerType.*
+import com.amazon.ion.impl.bin.IonRawBinaryWriter_1_1.ContainerType.List
```

Comment:
🗺️ IntelliJ was getting confused between `kotlin.List` (which is implicitly imported) and`ContainerType.List`, so I'm explicitly importing this to make IntelliJ happy. When I go to clean things up later, I'll switch the `ContainerType` enum variants to screaming snake case to avoid this problem.